src/clientSideScene/sceneEntities.ts

```diff
@@ -549,10 +565,101 @@ export class SceneEntities {
       ...mouseEnterLeaveCallbacks(),
     })
   }
-  setupSketchIdleCallbacks = (pathToNode: PathToNode) => {
+  setupSketchIdleCallbacks = ({
```

Comment:
This feels like it could hint at a state machine in the future! I like how you've implemented it

Reply:
Oh yeah, maybe.